gitops-gh-pr/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -l
+set -e
+
+# Configure git
+echo "https://gitops:$GITHUB_TOKEN@github.com" > ~/.git-credentials
+git config --local credential.helper store
+git config --local user.email "gitops@liatr.io"
+git config --local user.name "GitOps Automation"
+
+# Commit change
+MESSAGE="gitops: $COMMIT_MESSAGE"
+git add .
+git commit -m "$MESSAGE"
+
+# Create branch
+BRANCH="gitops-$(git rev-parse --short HEAD)"
+git checkout -b $BRANCH
+git push origin $BRANCH
+
+# Create pull request
+gh pr create --fill
```

Comment:
Should this exit if a PR is already open for the gitops branch? If a release was recreated, the PR made by the previous run is still valid, but I think the gh cli would exit with a non-zero code. 

I know we ran into something similar trying to do GitOps in ADO.